src/myuw-help.js

```diff
@@ -90,7 +107,10 @@ class MyUWHelp extends HTMLElement {
    * @param {string} newState Optional parameter, either 'open' or 'closed'
    */
   setDialogState(newState) {
-    switch(newState) {
+    let firstTabFocus;
```

Comment:
`let` confused me a bit here, making me think there were multiple assignments, but it looks like there's only one, and no need for mutability.
Consider simplifying to:
```javascript
const firstTabFocus = this.focusableElements.length ? this.focusableElements[0] : this.$dialogCloseButton;
```

Reply:
This logic has changed, and I'll keep it as let.